### PR TITLE
app-admin/passwordsafe-1.08_beta version bump

### PR DESCRIPTION
**Pull request by lorem-ipsum**
This bumps passwordsafe to the latest release. This also fixes the build failure present in the previous version reported at [#653698](https://bugs.gentoo.org/show_bug.cgi?id=683698).

### CODE REVIEW

File: app-admin/passwordsafe/passwordsafe-1.08_beta.ebuild [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit eutils flag-o-matic wxwidgets cmake-utils
```
Comment (a17r) [inline app-admin/passwordsafe/passwordsafe-1.08_beta.ebuild]: as far as I can see you don't need `eutils` here, you just use functions from `desktop.eclass`

**Comment by lorem-ipsum** [comment]
Thanks for the review, I've implemented the changes.
Can I also drop the check for c++11? For the 17.0 profiles this always works as they require c++14, are the olders still supported?

**Comment by a17r** [comment]
> Can I also drop the check for c++11? For the 17.0 profiles this always works as they require c++14, are the olders still supported?

Just put it on todo for next bump.

**Comment by lorem-ipsum** [comment]
Ok, will do. Thank you very much.

**Pull request closed without merging**